content/lua/concepts/mathematical-library/terms/max/max.md

```diff
@@ -0,0 +1,78 @@
+---
+Title: 'max()'
+Description: 'Used to return the maximum value among a set of numerical input values.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+Lua's **`math.max()`** function is a standard function that is a part of the `math` library, and it is the best solution for determining the maximum value from a list of numerical inputs, regardless of the number of values in this list. Whether the task is just simple arithmetic or a complex mathematical algorithm, `math.max()` is a versatile and fundamental helper function for a variety of cases.
+
+> **Note:** `math.min()` is the opposing function, which will return the minimum value among a set of numerical values.
+
+## Syntax
+
+Since `max()` is a method from Lua's library `math`,  it is called as `math.max()`.
```

Comment:
```suggestion
Since `max()` is a method from Lua's library `math`, it is called as `math.max()`.
```
You need to make this change in order for us to merge.

Reply:
@Khalid-debugg also, after making this change make sure the branch is up-to-date with main.